File: Core/Entities/Book.cs
namespace Core.Entities
{
    public class Book : BaseEntity
    {
        public string Name { get; set; }
        public string Author { get; set; }
        public string Article { get; set; }
        public int PublishingYear { get; set; }
        public int Count { get; set; }
    }
}

File: client/src/app/books/book.ts
export interface Book {
    id: number;
    name: string;
    author: string;
    article: string;
    publishingYear: number;
    count: number;
    readers: Reader[];
  }

export interface Reader {
    id: number;
    firstName: string;
    lastName: string;
    middleName: string;
    dateOfBirth: string;
    books: null[];
  }